doc/ecmascript.txt

```diff
@@ -21,12 +21,6 @@ acceptable documentation, is actively developed and ought to work.
 Ok, so how to get the ECMAScript support working?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Some systems come with either the SpiderMonkey installed or as an option. It
-would be good to test if you can take the easy path and let the system take
-care of installation through a package system. Below are listed instructions
-on what package you need to install on various systems (please help improve
-the list). If all goes well you can proceed to rebuilding ELinks.
-
 On Debian testing (Etch) or unstable (SID), run the following:
 
 	$ apt-get install libmozjs-dev
```

Comment:
As I don't use Debian/ Ubuntu I have no idea if these packages work. I only tested this on Arch. I just left this as is.